src/epython/poke/processors.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
This was a helper module, and placeholder to be able to do api testing easier. 

Imagine an abstraction for different HTTP status codes that have different behavior and validation techniques, or capturing logs from a specific server when a specific request has been made.